bindings/rust/src/bindings/mod.rs

```diff
@@ -211,6 +273,61 @@ impl Blob {
     pub fn from_hex(hex_str: &str) -> Result<Self, Error> {
         Self::from_bytes(&hex_to_bytes(hex_str)?)
     }
+
+    pub fn samples_and_proofs(
+        &self,
+        kzg_settings: &KZGSettings,
+    ) -> Result<Vec<(Sample, KZGProof)>, Error> {
+        let samples_len = kzg_settings.max_width as usize;
```

Comment:
I like to refer to this as "data" instead of "samples". These are the not-yet chunked samples.